float32 value1
float32 value2
float32 value3
float32 value4
